pkg/machines/components/stateIcon.scss

```diff
@@ -3,12 +3,16 @@
 
   // In definition lists (for Overview),
   // match text size and alignment
-  .pf-c-description-list__description & {
+  .pf-c-table &,
```

Comment:
@garrett With the CSS custom code for labels in the tables.

![Screen Shot 2020-10-28 at 14 19 32](https://user-images.githubusercontent.com/14921356/97441155-b02c1f80-1928-11eb-88c6-853d3fce4167.png)

Without any custom CSS for labels in the tables.
![Screen Shot 2020-10-28 at 14 19 17](https://user-images.githubusercontent.com/14921356/97441160-b0c4b600-1928-11eb-8719-f2d73f3ad36e.png)

It seems to me super close, why do we need it?

Reply:
Ok I actually see why we need the margin-left: calc(var(--pf-c-label--PaddingLeft)*-1);

But all the rest?

Reply:
1. In the list, as seen above, it's not completely necessary (but doesn't hurt)
2. In the mobile version of the list, things do not align without it
3. In the details page, things do not align without it